### DISCUSSION TITLE
Using Pydantic 2 with Dagster

### DISCUSSION THREAD

**Discussion by yuval-airis** [dagster-io/dagster#16839] [Q&A] [answered]
I wanted to use Dagster in the same project I have [pydantic](https://docs.pydantic.dev/) in, but discovered that dagster only allows using pydantic < 2. (While I wanted to use the latest)

What causes this limitation? Is there any way to resolve it?

**Comment by DennisSchwartz** [comment]
Also facing this problem. Any updates would be appreciated! 🙏 :)

**Comment by rexledesma** [comment, marked as answer]
In October 23, 2023, [Dagster 1.5.5](https://github.com/dagster-io/dagster/releases/tag/1.5.5) was released with Pydantic 1 and 2 support. Please ensure that you're using the latest Dagster version if you want to use Pydantic 2.

If you have any issues with this, feel free to chime in on the tracking issue: https://github.com/dagster-io/dagster/issues/17612.

**Reply by DennisSchwartz** [reply]
Amazing! Thank you very much :)
